src/components/DateInput/DateInputPopover/index.jsx

```diff
@@ -0,0 +1,300 @@
+import React, { useRef, useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { Manager, Reference, Popper } from 'react-popper';
+import { CSSTransition } from 'react-transition-group';
+import FDS from 'lib/dictionary/js/styleConstants';
+import { isNotRefsEvent } from 'components/util/events';
+import { getPopperPlacement } from './util';
+import { useDisableScroll, useCloseOnScroll } from './hooks';
+
+export const VALID_POSITIONS = ['auto', 'top', 'right', 'bottom', 'left'];
+export const VALID_ALIGNMENTS = ['start', 'end', 'center'];
+export const VALID_INTERACTION_MODES = ['hover', 'click', 'controlled'];
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
+const Popover = React.forwardRef(
```

Comment:
This Popover is now `DateInputPopover` as that is the only thing in our codebase that is using it. When you start working on a breaking change for this component, it would be very easy to pull out this popover and just give DateInput it's own simple tippy instance.